sentry_sdk/integrations/aws_lambda.py

```diff
@@ -158,19 +228,22 @@ def inner(*args, **kwargs):
             )
 
 
-def _make_request_event_processor(aws_event, aws_context):
-    # type: (Any, Any) -> EventProcessor
+def _make_request_event_processor(aws_event, aws_context, configured_timeout):
+    # type: (Any, Any, Any) -> EventProcessor
     start_time = datetime.now()
 
     def event_processor(event, hint, start_time=start_time):
         # type: (Event, Hint, datetime) -> Optional[Event]
         extra = event.setdefault("extra", {})
+        remaining_time_in_milis = aws_context.get_remaining_time_in_millis()
+        exec_duration = configured_timeout - remaining_time_in_milis
```

Comment:
Is this guaranteed to be correct for every execution of the Lambda? Cold and warm starts?

Reply:
We are determining the "configured" timeout by calling `get_remaining_time_in_millis` at the start of the lambda function, so this probably already excludes cold startup overhead.